Prace_domowe/Praca_domowa2/Wolny_Krzysztof/PD2.ipynb

```diff
@@ -0,0 +1,1504 @@
+{
```

Comment:
<p>To jest std kolumny, a nie std z błędu</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/95/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Reply:
<p>I liczysz rmse dla całej kolumny, a nie dla imputowanych wartości</p><p> </p><p> </p><p> </p>